18F/dol-whd-14c
# Update web dependencies

#### What's this PR do?

Updates web project dependencies, as recommended by [Gemnasium](https://gemnasium.com/github.com/18F/dol-whd-14c).  Removes the ones that don't appear to be used.

#### Where should the reviewer start?

Check the dependencies to make sure the removed ones aren't needed somewhere.

#### How should this be manually tested?

Build and make sure everything still works.  Run automated tests to make sure they all still pass.

#### Any background context you want to provide?

The QASP requires that Gemnasium or similar static scanning SaaS give an all-clear on security.  This PR is to help us take a step in that direction.
